pom.xml

```diff
@@ -608,9 +608,9 @@
              because we will be able to change the configuration without re-compiling the configuration file
              (which is located in a dependency of the integration tests modules).
          -->
+        <test.elasticsearch.distribution>elastic</test.elasticsearch.distribution>
+        <test.elasticsearch.version>${version.org.elasticsearch.latest-8.4}</test.elasticsearch.version>
```

Comment:
As far as I can tell, you're only using this version in the POM, right?

Could you rename `version.org.elasticsearch.latest-8.4` to just `version.org.elasticsearch.latest`, `version.org.opensearch.latest-2.3` to `version.org.opensearch.latest`, and remove all other `-latest` properties? Since we'll maintain that in the Jenkinsfile from now on.